### PR TITLE
Add "exclude_paths" parameter to exclude unwanted files/directories

### LINKED ISSUE #609: Possibility to exclude directory via CLI + config
Everything's in title.
Just a reminder of why I ask it, it's because tests and js files are in my code directory. Maybe adding a File mask could be a great option too

### PR DESCRIPTION
**Pull request by TomasVotruba**
Closes #609

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Mergin because #612 depends on it and I need to add some changes there

Still feel free to review

**Comment by vasilvestre** [comment]
yeah, acutally unable but kater

File: README.md [hunk 1]
```diff
@@ -64,6 +64,15 @@ parameters:
         - '%kernel.project_dir%/vendor/project-without-composer'
 ```
 
+You can also exclude files or directories - use regex or [fnmatch](http://php.net/manual/en/function.fnmatch.php):
```
Comment (carusogabriel) [inline README.md]: We can simplify this link:
```diff
-http://php.net/manual/en/function.fnmatch.php
+https://php.net/fnmatch
```

Comment (TomasVotruba) [reply README.md]: Might be, but that extra manual work.
I simply copy what's in the URL on the site.

**Pull request merged**